Composer/packages/client/src/pages/publish/createPublishTarget.tsx

```diff
@@ -49,7 +49,7 @@ const CreatePublishTarget: React.FC<CreatePublishTargetProps> = (props) => {
     if (!newName || newName.trim() === '') {
       setErrorMsg(formatMessage('Must have a name'));
     } else {
-      const exists = !!props.targets?.find((t) => t.name.toLowerCase() === newName?.toLowerCase);
+      const exists = props.targets?.some((t) => t.name.toLowerCase() === newName?.toLowerCase);
```

Comment:
`t.name.toLowerCase() === newName?.toLowerCase` pretty sure this is a bug. Should be `newName?.toLowerCase()`